### PR TITLE
update redis to v7

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: Simon L <szaimen@e.mail.de>

### CODE REVIEW

File: Containers/redis/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # From https://github.com/docker-library/redis/blob/master/6.2/alpine/Dockerfile
-FROM redis:6.2.8-alpine
+FROM redis:7.0.7-alpine
```
Comment (szaimen) [inline Containers/redis/Dockerfile:2]: wait for alpine 3.17?

File: Containers/redis/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # From https://github.com/docker-library/redis/blob/master/6.2/alpine/Dockerfile
```
Comment (Zoey2936) [inline Containers/redis/Dockerfile:1]:
```suggestion
# From https://github.com/docker-library/redis/blob/master/7.0/alpine/Dockerfile
```

File: Containers/redis/Dockerfile (hunk 1 above)
Comment (Zoey2936) [reply Containers/redis/Dockerfile:2]: Maybe also pin the alpine version?
```suggestion
FROM 7.0.7-alpine3.17
```

Comment (szaimen) [reply Containers/redis/Dockerfile:2]: thought about this as well, however there was a time when minor posgres updates where not pushed with older alpine version anymore. That is why I removed that. Also it is pretty unlikely that alpine updates break this specific container.

**Review by Zoey2936 — APPROVED** [review]

**Pull request merged**